src/patternfly/components/Drawer/drawer.scss

```diff
@@ -460,6 +459,7 @@ $pf-c-drawer__panel--list--width: (25, 33, 50, 66, 75, 100);
   .pf-c-drawer.pf-m-panel-bottom {
     --pf-c-drawer--m-expanded__panel--BoxShadow: var(--pf-c-drawer--m-expanded--m-panel-bottom__panel--BoxShadow);
     --pf-c-drawer__panel--MaxHeight: 100%;
+    --pf-c-drawer__panel--FlexBasis--min: var(--pf-c-drawer__splitter--Height);
```

Comment:
Can you create a variable for the value used here, except move it up to the top of the component where the variables are set?

```
--pf-c-drawer--m-panel-bottom__panel--FlexBasis--min: var(--pf-c-drawer__splitter--Height);
```

And then here on this line, change it to:

```
--pf-c-drawer__panel--FlexBasis--min: var(--pf-c-drawer--m-panel-bottom__panel--FlexBasis--min);
```

That will create a variable that users can modify (`--pf-c-drawer--m-panel-bottom__panel--FlexBasis--min`) in their application by setting the code below.

```
.pf-c-drawer {
  --pf-c-drawer--m-panel-bottom__panel--FlexBasis--min: // some custom value;
}
```

Otherwise they would have to set it via:

```
.pf-c-drawer.pf-m-panel-bottom { 
  --pf-c-drawer__panel--FlexBasis--min: // some custom value;
}
```